pkg/nodebootstrap/userdata.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"strings"
 
+	gfn "github.com/awslabs/goformation/cloudformation"
```

Comment:
I see what you wanted to do here, but actually it's better to avoid introducing package dependencies like this.

Reply:
it is dependant on `gfn` just not explicitly, returning string was not helpful, so I've changed it to explicit type instead of string-wrapped one ;)